Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -131,11 +131,22 @@ public final class PledgeCTAContainerViewViewModel: PledgeCTAContainerViewViewMo
 // MARK: - Functions
 
 private func pledgeCTA(project: Project, backing: Backing?) -> PledgeStateCTAType {
+  let uuid = (AppEnvironment.current.device.identifierForVendor
+    ?? UUID()).uuidString // TODO: move to environment
+  let optimizelyVariant = AppEnvironment.current.optimizelyClient?
+    .variant(for: OptimizelyExperiment.Key.pledgeCTACopy, userId: uuid)
+
   guard let projectBacking = backing, project.personalization.isBacking == .some(true) else {
     if currentUserIsCreator(of: project) {
       return PledgeStateCTAType.viewYourRewards
     }
 
+    if let variant = optimizelyVariant,
```

Comment:
This can be slightly simplified as:
```swift
if OptimizelyExperiment.Variant.experimental == optimizelyVariant, project.state == .live {
  return PledgeStateCTAType.seeRewards
}
```